applications/MACI-CDao.md

```diff
@@ -0,0 +1,268 @@
+| 1.     | Usage example  |  Example of how to use the pallet inside code  |    
+
+
+### Milestone 2  Pallet logic — Auxiliary functions features
+
+- **Estimated Duration:** 1 month
+- **FTE:**                1.5
+- **Costs:**              10,000 USD 
+- Description: Pallet written in Rust for Substrate 2.0 based blockchains. The Pallet will implement  member management and bit commitment voting process,together with decentralized decision pallets, it forms the Minimum anti-collusion Infrastructure (MACI).   
+
+| Number | Deliverable    | Specification | 
+| ------ | -----------    | ------------- |
+| 0a.    | License        | Apache 2.0    |  
+| 0b.    | Documentation  | Code documentation, API reference |  
+| 0c.    | test           | The code will have unit-test coverage (min.70%) to ensure functionality and robustness. In the guide, we will describe how to run these tests. |  
+| 1.     |  Usage example | Example of how to use the pallet inside code.|   
+| 2.     | anti-collusion | Implement a mechanism to prevent bribery and collusion, leveraging encrypting votes potentially via Minimum anti-collusion Infrastructure (MACI) | 
+
+### Milestone 3  Demo case —— Extended functionality features
+
+- **Estimated Duration:** 1 month
+- **FTE:**                2
+- **Costs:**              10,000 USD 
+  
+- Description: Develop a simple ddapp, demonstrate the minimum anti-collusion mechanism, and publish an article that explains the work done as part of the grant   
+
+| Number | Deliverable    | Specification | 
+| ------ | -----------    | ------------- |
+| **0a.**| License        | Apache 2.0    |  
+| **0b.**| Documentation  | Code documentation, API reference. |  
+| **0c.**| test           | The code will have unit-test coverage (min. 70%) to ensure functionality and robustness.| 
+| **0d.**| Article        | We will publish an article that explains the work done as part of the grant. |  
+| 1.     | Anti-collusion | Implement Minimum Anti-Collusion Infrastructure (MACI). | 
+| 2.     | Voting Example | Integrate a basic voting example that test the mechanism.  | 
+   
+
+
+## Future Plans 
+
+ First, we hope to extend the minimum anti-collusion infrastructure into a new consensus mechanism and build a super fair governance chain;\\
+ Second, we will construct a general, super fair solution to the general dispute problem, i.e., a sufficient anti-collusion infrastructure;\\
+ Finally, we look forward to the emergence of more and more decentralized decision-making applications (ddapps) on the super fair governance chain, which requires the joint efforts of community members. 
+
+## Additional Information : 
+
+**How did you hear about the Grants Program?** 
+
+ Announcement by  OneBlock. 
+ 
+**other additional information**  
+
+- The paper about super fair governance is still under review. 
```

Comment:
if this is the only source available for this cake cutting model then please forward it to us at grants@web3.foundation already so we can take a look :)

Reply:
Thank you very much for your interest. The previous journal did not consider it suitable for their goals. Because this paper involves several fields, it is not easy to find a suitable journal. Currently, I am modifying the format of the paper to adapt to the new target journal. After this work is completed, I will send this email and look forward to your correction.

Reply:
Okay, please ping me then so I can follow up :) Thanks!

Reply:
Thank you very much for your attention.Since the paper involves several fields such as information processing, economics, and social sciences, the original thesis was written in the style of computer science. Now it seems that it may lean more towards economics and social sciences. Therefore, there are significant changes in the content of the paper. I will strive to reorganize the thesis by this weekend. So, please wait a few days, and I will deal with this issue.